darkside-tests/src/utils.rs

```diff
@@ -384,3 +384,54 @@ impl TreeState {
         })
     }
 }
+
+pub async fn init_darksidewalletd() -> Result<(DarksideHandler, DarksideConnector), String> {
+    let handler = DarksideHandler::default();
+    let server_id = zingoconfig::construct_lightwalletd_uri(Some(format!(
+        "http://127.0.0.1:{}",
+        handler.grpc_port
+    )));
+    let connector = DarksideConnector(server_id);
+    let mut client = connector.get_client().await.unwrap();
+    // Setup prodedures.  Up to this point there's no communication between the client and the dswd
+    client.clear_address_utxo(Empty {}).await.unwrap();
+
+    // reset with parameters
+    connector
+        .reset(1, String::from(BRANCH_ID), String::from("regtest"))
+        .await
+        .unwrap();
+
+    // stage genesis block
+    connector
+        .stage_blocks_stream(vec![String::from(crate::constants::GENESIS_BLOCK)])
+        .await?;
+    connector
+        .add_tree_state(constants::first_tree_state())
+        .await
+        .unwrap();
+
+    Ok((handler, connector))
+}
+pub async fn stage_transaction(connector: &DarksideConnector, height: u64, hex_transaction: &str) {
```

Comment:
This function stages a transaction and updates the tree state. is this something that will be done for every transaction? we probably a name that indicates that the transaction staged will also alter a tree state at height `height`. 

If this later action of updating the tree state is something that some tests will not use we probably want this to be wrapped on a flag to pass to the function. 

like `fn stage_transaction(connector: &DarksideConnector, height: u64, hex_transaction: &str, updating_tree_state: Bool)`